stock_quant_package_product_packaging/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Camptocamp SA
```

Comment:
```suggestion
# Copyright 2020 Camptocamp SA
```

and idem for others

Reply:
Again, not need as there were no changes this year. :wink: